CHANGELOG.md

```diff
@@ -6,6 +6,33 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+## [6.2.0] - 2021-03-23
+
+### Added
+
+- Adds the Notification Controller (to support "what's new" type announcments in-app) ([#329](https://github.com/MetaMask/controllers/pull/329))
+
+### Changed
+
+- Explicitly add ethereumjs-tx as a package.json dependency ([#392](https://github.com/MetaMask/controllers/pull/392))
+- Add `types` manifest field to package.json ([#391](https://github.com/MetaMask/controllers/pull/391))
+- Use "options bag" for parameters for BaseControllerV2 constructor ([#388](https://github.com/MetaMask/controllers/pull/388))
+- Ensure `uuid` dependency is type-checked ([#403](https://github.com/MetaMask/controllers/pull/403))
+- Update TypeScript to v4.2 ([#369](https://github.com/MetaMask/controllers/pull/369))
+- Asset metadata type conditionally requires error field, disallows for non-errors ([#395](https://github.com/MetaMask/controllers/pull/395))
+- Consistently use BN type for token balances ([#398](https://github.com/MetaMask/controllers/pull/398))
+- `approveTransaction` of TransactionController gets nonce from txMeta, falls back to network nonce ([#381](https://github.com/MetaMask/controllers/pull/381))
```

Comment:
This doesn't seem like a good description :thinking: This commit added support for custom nonces. This should probably go under "Added" as well, since it's a new feature.

```suggestion
- Add support for specifying a custom nonce ([#381](https://github.com/MetaMask/controllers/pull/381))
```

Reply:
I noticed that it's still under "Changed" - what do you think about the suggestion to recategorize it as 'Added"?